lib/components/account_action_row.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/components/profile_avatar.dart';
+import 'package:seeds/design/app_theme.dart';
+
+class AccountActionRow extends StatelessWidget {
```

Comment:
We have too many rows that look very similar. 

We need to make a WELL done global row for user and reuse. 

Ill reuse this row in future prs.

Reply:
Yeah, this looks like the `MemberRow` global component, not sure if there are others similar.